code/modules/reagents/Chemistry-Reagents/Other-Reagents.dm

```diff
@@ -333,13 +333,39 @@
 			M.AdjustWeakened(2)
 	..()
 
+	if(reac_volume > 11)
+		metabolization_rate = 1 * REAGENTS_METABOLISM
+
+		if(istype(M, /mob/living/carbon/human))
```

Comment:
This is a pretty neat thing, what if we put it into it's own human/proc() so that it can used elsewhere? Possibly a disease that turns people in guidos